loops-arrays-objects.js

```diff
@@ -31,10 +29,22 @@ const scores = [
 // A
 // D
 // ==========================================
-
-
-
-
+//
+
+for (let i = 0; i <scores.length; i++) {
+    if (scores[i].score < 100) {
+        scores[i].grade = 'A';
+    } if (scores[i].score < 90) {
+        scores[i].grade = 'B';
+    } if (scores[i].score < 80) {
+        scores[i].grade = 'C';
+    } if (scores[i].score < 70) {
+        scores[i].grade = 'D';
+    } if (scores[i].score < 60) {
+        scores[i].grade= 'F';
+    }
```

Comment:
Ik zou de volgorde van if's omdraaien (van 60 naar 100) en `else if` gebruiken. Dan wordt bij een score van 59 maar één keer `scores[i].grade` gezet in plaats van 5x 😉